init.sh

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env sh
 
+function install_docker () {
+    echo "Installing Docker. Press ctrl-c to quit"
+    # Give the user time to respond
```

Comment:
Rather than a period to respond, we should bring the script execution to a full stop until the user has either consented or cancelled. I'm picturing something like this:
https://www.shellhacks.com/yes-no-bash-script-prompt-confirmation/
or for more ideas: https://stackoverflow.com/questions/1885525/how-do-i-prompt-a-user-for-confirmation-in-bash-script

Reply:
That is a great point!

Heh, just noticed the JavaScript syntax error I threw in there. Whoops. Just committed a fix for that